aws-throwaway/src/backend/cli/instance_type.rs

```diff
@@ -0,0 +1,3932 @@
+            InstanceType::Z1d12xlarge => "z1d.12xlarge",
+            InstanceType::Z1d2xlarge => "z1d.2xlarge",
+            InstanceType::Z1d3xlarge => "z1d.3xlarge",
+            InstanceType::Z1d6xlarge => "z1d.6xlarge",
+            InstanceType::Z1dLarge => "z1d.large",
+            InstanceType::Z1dMetal => "z1d.metal",
+            InstanceType::Z1dXlarge => "z1d.xlarge",
+            InstanceType::Unknown => "unknown",
+        }
+    }
+    /// Returns all the `&str` representations of the enum members.
+    pub const fn values() -> &'static [&'static str] {
+        &[
+            "a1.2xlarge",
+            "a1.4xlarge",
+            "a1.large",
+            "a1.medium",
+            "a1.metal",
+            "a1.xlarge",
+            "c1.medium",
+            "c1.xlarge",
+            "c3.2xlarge",
+            "c3.4xlarge",
+            "c3.8xlarge",
+            "c3.large",
+            "c3.xlarge",
+            "c4.2xlarge",
+            "c4.4xlarge",
+            "c4.8xlarge",
+            "c4.large",
+            "c4.xlarge",
+            "c5.12xlarge",
+            "c5.18xlarge",
+            "c5.24xlarge",
+            "c5.2xlarge",
+            "c5.4xlarge",
+            "c5.9xlarge",
+            "c5.large",
+            "c5.metal",
+            "c5.xlarge",
+            "c5a.12xlarge",
+            "c5a.16xlarge",
+            "c5a.24xlarge",
+            "c5a.2xlarge",
+            "c5a.4xlarge",
+            "c5a.8xlarge",
+            "c5a.large",
+            "c5a.xlarge",
+            "c5ad.12xlarge",
+            "c5ad.16xlarge",
+            "c5ad.24xlarge",
+            "c5ad.2xlarge",
+            "c5ad.4xlarge",
+            "c5ad.8xlarge",
+            "c5ad.large",
+            "c5ad.xlarge",
+            "c5d.12xlarge",
+            "c5d.18xlarge",
+            "c5d.24xlarge",
+            "c5d.2xlarge",
+            "c5d.4xlarge",
+            "c5d.9xlarge",
+            "c5d.large",
+            "c5d.metal",
+            "c5d.xlarge",
+            "c5n.18xlarge",
+            "c5n.2xlarge",
+            "c5n.4xlarge",
+            "c5n.9xlarge",
+            "c5n.large",
+            "c5n.metal",
+            "c5n.xlarge",
+            "c6a.12xlarge",
+            "c6a.16xlarge",
+            "c6a.24xlarge",
+            "c6a.2xlarge",
+            "c6a.32xlarge",
+            "c6a.48xlarge",
+            "c6a.4xlarge",
+            "c6a.8xlarge",
+            "c6a.large",
+            "c6a.metal",
+            "c6a.xlarge",
+            "c6g.12xlarge",
+            "c6g.16xlarge",
+            "c6g.2xlarge",
+            "c6g.4xlarge",
+            "c6g.8xlarge",
+            "c6g.large",
+            "c6g.medium",
+            "c6g.metal",
+            "c6g.xlarge",
+            "c6gd.12xlarge",
+            "c6gd.16xlarge",
+            "c6gd.2xlarge",
+            "c6gd.4xlarge",
+            "c6gd.8xlarge",
+            "c6gd.large",
+            "c6gd.medium",
```

Comment:
Did you pull this from a dump of all the instance types?

Reply:
yep, I copied its the InstanceType copied directly from aws-sdk-ec2